VHostScan/lib/helpers/color_helper.py

```diff
@@ -0,0 +1,104 @@
+from termcolor import colored
+
+__COLOR_ENABLED = True
+
+text_settings = {
+    'process':      ('cyan', []),
+    'result_head':  ('magenta', []),
+    'result':       ('white', []),
+    'key':          ('yellow', []),
+    'link':         ('green', ['underline']),
+    'error':        ('red', ['bold']),
+    'warning':      ('red', ['bold', 'dark']),
```

Comment:
dark is only supported on two terminals according to termcolor's page. We probably shouldn't rely on it.

Reply:
I considered that and I think it will automatically omit the 'dark' attribute on most the terminals which is still look fine, just a little bit weaker. However, I am not sure if this is a good thinking. What is your word?

Reply:
Ok, I just tested dark on both ubuntu and kali, and it seems to be working on both of them, so I guess we should keep it the way you have it.